internal/network/command_dns_forwarder_update.go

```diff
@@ -0,0 +1,91 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingdnsforwarderv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking-dnsforwarder/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newDnsForwarderUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "update <id>",
+		Short:             "Update an existing DNS forwarder.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validDnsForwarderArgs),
+		RunE:              c.dnsForwarderUpdate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Update the name of DNS forwarder "dnsf-123456".`,
+				Code: "confluent network dns forwarder update dnsf-123456 --name my-new-dns-forwarder",
+			},
+			examples.Example{
+				Text: `Update the DNS server IPs and domains of DNS forwarder "dnsf-123456".`,
+				Code: "confluent network dns forwarder update dnsf-123456 --dns-server-ips 10.200.0.0,10.201.0.0 --domains abc.com,def.com",
+			},
+		),
+	}
+
+	cmd.Flags().String("name", "", "Name of the DNS forwarder.")
+	cmd.Flags().StringSlice("domains", nil, "A comma-separated list of domains for the DNS forwarder to use.")
+	cmd.Flags().StringSlice("dns-server-ips", nil, "A comma-separated list of IP addresses for the DNS server.")
```

Comment:
It looks like these flags are used in `create` and `update`, so it might be good to put them in a functions (like the `addConfigFlag` function).